pkg/crypto/crypto.go

```diff
@@ -5,14 +5,17 @@ import (
 	"io"
 	"io/ioutil"
 	"strings"
-
+	
 	"github.com/pkg/errors"
 	"golang.org/x/crypto/openpgp"
 	"golang.org/x/crypto/openpgp/armor"
 )
 
 const (
 	wmcoMarker = "<wmcoMarker>"
+	startTag = "-----BEGIN ENCRYPTED DATA-----"
+	endTag   = "-----END ENCRYPTED DATA-----"
+	newLine = "\n"
```

Comment:
I'd prefer not to make this is a constant, to me it would be the same as defining `a` or `b` as a constant